docs/howto/manage-models.md

```diff
@@ -3,10 +3,11 @@
 
 > See also: {ref}`model`
 
+(add-a-model)=
 ## Add a model
 
 ```{caution}
-**If you have multiple credentials:** Be careful which one you use for the new model. Any machines subsequently on the model will be associated with this credential. As such, make sure you're not spending resources for the wrong cloud account! 
+**If you have multiple credentials:** Be careful which one you use for the new model. Any machines subsequently on the model will be associated with this credential. As such, make sure you're not spending resources for the wrong cloud account!
```

Comment:
"Any machines added to the model subsequently".

Reply:
Fixed, thanks!